src/templates/template_map.cpp

```diff
@@ -139,9 +141,21 @@ void TemplateMap::drawTemplate(QPainter* painter, const QRectF& clip_rect, doubl
 	}
 	
 	RenderConfig::Options options;
+	auto scaling = scale;
 	if (on_screen)
+	{
 		options |= RenderConfig::Screen;
-	RenderConfig config = { *(template_map.get()), transformed_clip_rect, scale, options, qreal(opacity) };
+		scaling = Util::mmToPixelPhysical(scale);
+	}
+	else
+	{
+		auto dpi = painter->device()->physicalDpiX();
+		if (!dpi)
+			dpi = painter->device()->logicalDpiX();
+		if (dpi > 0)
+			scaling = dpi / 25.4;
```

Comment:
`… *= …`

Reply:
The only question left at this place is what is the exact meaning of `RenderConfig.scaling` and what is its unit of measurement. Without this information it is difficult to tell what is the appropriate   default for `scaling` in case the painter device does not provide DPI information. However, if you don't know the answer from top of your head, feel free to disregard the question and push the change into `master` as it indeed resolved the issue at hand.

Reply:
Well, I already found the answer, and I added it to the documentation now.